.github/workflows/add_label_from_comment.yml

```diff
@@ -74,3 +74,70 @@ jobs:
           curl --request DELETE \
             --url https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.issue.number }}/labels/awaiting-author \
             --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}'
+
+  add_delegated_label:
+    name: Add delegated label
+    if: (toJSON(github.event.issue.pull_request) != 'null') && (startsWith(github.event.comment.body, 'bors d') || contains(toJSON(github.event.comment.body), '\r\nbors d')
```

Comment:
Perhaps we don't care, but won't this misfire on `bors d-`?

Reply:
Oh, good point. I'll just write out the different cases.

Reply:
Actually, it seems that `bors d-` isn't a command: https://bors.tech/documentation/ https://github.com/bors-ng/bors-ng/blob/a1b31786d48839f67fd5691564217e8538f74b24/lib/web/command.ex